examples/neutron.rs

```diff
@@ -0,0 +1,187 @@
+use localic_utils::{
+    types::contract::{
+        AuctionStrategy, ChainHaltConfig, MinAmount, PairType, PriceFreshnessStrategy,
+    },
+    ConfigChainBuilder, TestContextBuilder,
+};
+use std::error::Error;
+
+/// Demonstrates using localic-utils for neutron.
+fn main() -> Result<(), Box<dyn Error>> {
+    env_logger::init();
+
+    let neutron = ConfigChainBuilder::default()
```

Comment:
in the future we will have many chains being built that way and vast majority of them will stick to default values. I think it would be nice to have default chain config builder calls for neutron, stride, osmosis, etc. Perhaps `::default_neutron().build()` or something along the lines.

Reply:
Good idea!

Reply:
Implemented this!